db/migrate/20160425161234_remove_deleted_tables_from_replication_settings.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveDeletedTablesFromReplicationSettings < ActiveRecord::Migration[5.0]
+  class SettingsChange < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :value
+  end
+
+  def up
+    deleted_tables = %w(miq_events miq_license_contents vim_performances)
```

Comment:
I only really mention because otherwise the migration doesn't need to be backported at all, which makes it less critical (that is, if you have only extra tables in the list, then it doesn't hurt anything, but if you are *missing* tables in the list due to a rename, then that's a real problem).